app/controllers/admin/users_controller.rb

```diff
@@ -142,7 +145,8 @@ def update
 
     def destroy
       @user.paper_trail_event = 'deactivate'
-      @user.update(active: false)
+      # update_column() allows us to update the user even if the record is invalid
+      @user.update_column(:active, false)
```

Comment:
Note; this would not trigger callbacks on user. Checked with Elliot and it sounded like there isn't anything we need to do when the user is deactivated